conf/app.routes

```diff
@@ -18,7 +18,10 @@ GET           /individuals/subcontractor/idms/taxpayer/:idKey
 
 
 POST          /individuals/debts/time-to-pay/quote                                                          uk.gov.hmrc.debttransformationstub.controllers.TimeToPayController.generateQuote
-POST          /debts/time-to-pay/quote                                                          uk.gov.hmrc.debttransformationstub.controllers.TimeToPayController.generateQuote
+POST          /debts/time-to-pay/quote                                                                      uk.gov.hmrc.debttransformationstub.controllers.TimeToPayController.generateQuote
+
+POST          /individuals/debts/time-to-pay/affordability/affordable-quotes                                uk.gov.hmrc.debttransformationstub.controllers.TimeToPayController.generateAffordabilityQuote
```

Comment:
`individuals/time-to-pay/affordability/affordable-quotes` not `/individuals/debts/time-to-pay/affordability/affordable-quotes` :)